common/coconut-interface/src/lib.rs

```diff
@@ -0,0 +1,170 @@
+pub use coconut_rs::*;
+use coconut_rs::{Attribute, Base58, BlindSignRequest, BlindedSignature, PublicKey};
+use getset::{CopyGetters, Getters};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Getters, CopyGetters, Clone)]
+pub struct Credential {
+    #[getset(get = "pub")]
```

Comment:
Is using separate library for getters/setters really worth it? At some point we will definitely want to limit our dependencies to bare minimum so I wonder if we couldn't just write those couple of lines ourselves.

Reply:
I kind of like these getters/setters too :D . Regarding limiting the dependencies, I'm thinking we should apply that to the actual `coconut-rs` crate, while this interface that we're using is not so sensitive.